doc/user/content/transform-data/patterns/durable-subscriptions.md

```diff
@@ -9,6 +9,8 @@ aliases:
   - /transform-data/patterns/time-travel-queries/#history-retention-period
 ---
 
+[//]: # "TODO: Move to Serve results section"
```

Comment:
The query optimization page could also probably go into the Serve results section as well ... but baby steps.

Reply:
Disagree! Query optimization applies to transformation first. We should discourage ad-hoc queries, in general, so optimization should never really happen at serving time.

Reply:
But (and things might have changed since the first 3 weeks) ... I was under the impression that while not specifically encouraging ad-hoc queries, I thought that it was a gap in our material that people didn't know that we're good with ad-hoc queries too.  But ... that can be fleshed out whenever I do the Serve results section.

Reply:
> people didn't know that we're good with ad-hoc queries too

We're good at serving ad-hoc queries that the underlying data model is optimized for, not at serving arbitrary ad-hoc queries. So, in the end, it's dependent on someone building the right foundation for the expected query patterns.